tests/conftest.py

```diff
@@ -3,15 +3,20 @@
 import httpretty
 import pytest
 from h_matchers import Any
+from pytest_factoryboy import register
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
+from tests.factories import TranscriptFactory
 from tests.factories.factoryboy_sqlalchemy_session import (
     clear_factoryboy_sqlalchemy_session,
     set_factoryboy_sqlalchemy_session,
 )
 from via.db import Base
 
+# Each factory has to be registered with pytest_factoryboy.
+register(TranscriptFactory)
```

Comment:
No way around this AFAICT. `pytest_factoryboy` comes with an `@register` decorator that you can just put on the factory classes where you define them, but it doesn't seem to work.